database/repository/audit/audit_test.go

```diff
@@ -99,7 +99,7 @@ func TestAudit(t *testing.T) {
 	}
 }
 
-func writeAudit(t *testing.T) {
+func writeAudit(t *testing.T) { //nolint
```

Comment:
Please be more specific with your nolint usage. Here is an example from our codebase:
`//nolint:forcetypeassert // Not necessary from a pool`
we specify what we are nolinting and we provide a comment on why we are nolinting